ros2_batch_job/__main__.py

```diff
@@ -356,6 +371,51 @@ def build_and_test(args, job):
     # return 0 if ret_test == 0 and ret_testr == 0 else 1
     return 0
 
+def build_with_sanitizer(cmd, args, job):
+    print('# BEGIN SUBSECTION: build with sanitizer')
+    sanitizer_type = args.enable_sanitizer_type
+    sanitizer_cmake_args = [
+    '-DCMAKE_CXX_FLAGS="-fsanitize=' + sanitizer_type + ' -g"',
```

Comment:
Maybe having some kind of shared flags between all jobs would be good. Not sure if necessary here.

Reply:
Used a single sanitizer flags variable for all 4 args. it might be tricky to have it across jobs though